src/main/drivers/timer_common.c

```diff
@@ -38,55 +38,61 @@ timerIOConfig_t *timerIoConfigByTag(ioTag_t ioTag)
     UNUSED(ioTag);
     return NULL;
 }
-#endif
 
 static uint8_t timerIndexByTag(ioTag_t ioTag)
 {
-#ifdef USE_TIMER_MGMT
     for (unsigned i = 0; i < MAX_TIMER_PINMAP_COUNT; i++) {
         if (timerIOConfig(i)->ioTag == ioTag) {
             return timerIOConfig(i)->index;
         }
     }
-#else
-    UNUSED(ioTag);
-#endif
     return 0;
 }
 
 const timerHardware_t *timerGetByTagAndIndex(ioTag_t ioTag, unsigned timerIndex)
 {
-    if (!ioTag) {
+
+    if (!ioTag || !timerIndex) {
         return NULL;
     }
 
 #if TIMER_CHANNEL_COUNT > 0
```

Comment:
This can be removed from this implementation - this was only added as a crutch to stop SITL (which has `TIMER_CHANNEL_COUNT == 0`) from throwing a warning.